torch/linalg/__init__.py

```diff
@@ -431,6 +431,55 @@
             [1, 2, 2, 2]])
 """)
 
+multi_dot = _add_docstr(_linalg.linalg_multi_dot, r"""
+linalg.multi_dot(tensors, *, out=None)
+
+Efficiently multiplies two or more matrices given by :attr:`tensors` by
+selecting the matrix chain multiplication order that results in the least
```

Comment:
by ordering the multiplications so that the fewest arithmetic operations are performed.